backend/src/v5/models/tickets.comments.js

```diff
@@ -58,22 +57,31 @@ TicketComments.addComment = async (teamspace, project, model, ticket, commentDat
 
 const getUpdatedComment = (oldComment, updateData) => {
 	const updatedAt = new Date();
-	const formattedComment = { updatedAt, ...updateData };
+	const formattedComment = { ...updateData, updatedAt };
 
 	const historyEntry = {
 		timestamp: oldComment.updatedAt,
 		...(oldComment.comment ? { comment: oldComment.comment } : {}),
 		...(oldComment.images ? { images: oldComment.images } : {}),
 	};
 
-	formattedComment.history = concat(oldComment.history ?? [], [historyEntry]);
+	formattedComment.history = [...(oldComment.history ?? []), historyEntry];
 
 	return formattedComment;
 };
 
 TicketComments.updateComment = async (teamspace, oldComment, updateData) => {
 	const formattedComment = getUpdatedComment(oldComment, updateData);
-	await updateOne(teamspace, { _id: oldComment._id }, { $set: { ...formattedComment } });
+
+	const toSet = { $set: { ...formattedComment } };
```

Comment:
I don't think there's much point calling having 2 variables here

just have it as a single object then you don't have to do `...` on L84